GoogleDataTransport/GDTCORLibrary/GDTCORReachability.m

```diff
@@ -98,22 +104,25 @@ - (instancetype)init {
       }
     });
   }
+#endif
   return self;
 }
 
-- (void)setCallbackFlags:(SCNetworkReachabilityFlags)flags {
+#if !TARGET_OS_WATCH
+- (void)setCallbackFlags:(GDTCORNetworkReachabilityFlags)flags {
   if (_callbackFlags != flags) {
     self->_callbackFlags = flags;
   }
 }
+#endif
 
 @end
 
-static void GDTCORReachabilityCallback(SCNetworkReachabilityRef reachability,
-                                       SCNetworkReachabilityFlags flags,
+#if !TARGET_OS_WATCH
```

Comment:
Same thing, you might not have to #if this out because of param changes.

Reply:
Add #pragma clang diagnostic to suppress unused function warning.